brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Ability to modify breakpoints for Dynamic CSS.

### Description

- Trello - https://trello.com/c/E6ObGGUe/143-ability-to-modify-breakpoints-for-dynamic-css
- Ability to modify breakpoints for Dynamic CSS.

### Types of changes




- New feature (non-breaking change which adds functionality)

### How has this been tested?

- Add breakpoint in wp-config.php
  define( 'UAGB_TABLET_BREAKPOINT', '800' );
  define( 'UAGB_MOBILE_BREAKPOINT', '500' );
- Apply some dynamic value eq: Font Size etc and check the value with the specified breakpoint by inspecting

### Checklist:
